src/hooks/useUpcomingReleases.tsx

```diff
@@ -0,0 +1,65 @@
+import { useEffect, useState } from 'react';
+import { useReleaseData } from './useReleaseHistory';
+
+export type ReleaseTypes = 'current' | 'lts' | 'maintenance' | 'endoflife';
```

Comment:
I'd like to have it as enum. Values will not spread through out the codebase

Reply:
Sure